firebase-firestore/src/main/java/com/google/firebase/firestore/core/FirestoreClient.java

```diff
@@ -143,6 +145,10 @@ public FirestoreClient(
           if (gcScheduler != null) {
             gcScheduler.stop();
           }
+
+          if (indexScheduler != null) {
```

Comment:
Should this scheduler always run? We may also need to backfill indices that were configured after client startup.

Reply:
In my mind, the scheduler should always run, but whether the scheduler schedules anything is another question. We can change this as necessary as we build more functionality out.